### PR TITLE
Make the progress table sortable

### PR DESCRIPTION
**Pull request by halkeye**
image

### CODE REVIEW

**Comment by halkeye** [comment]
I can't apparently add @zbynek as a reviewer (probably no access to infra) so comment tagging

**Comment by zbynek** [comment]
Could the status sorting be OK > PR > TODO? Otherwise looks good. As a nitpick I'm not sure if we need to depend on 5 external domains.

**Comment by halkeye** [comment]
> Could the status sorting be OK > PR > TODO?

I'll admit I never understand the math behind the sort method, -1, 0, 1

https://examples.bootstrap-table.com/#options/custom-sort.html#view-source says we can use that logic, I just don't know how to use it.

> As a nitpick I'm not sure if we need to depend on 5 external domains.

Your right, its a simple app, I'll just vendor and shove them in static.

**Comment by halkeye** [comment]
> I'll admit I never understand the math behind the sort method, -1, 0, 1

oooh, wonder if you could use indexof

['OK', 'PR', TODO'].indexof(b.status) - ['OK', 'PR', TODO'].indexof(a.status)

or something

File: reports.js [hunk 1]
```diff
@@ -39,39 +39,14 @@ async function pluginsReport() {
...
-      default:
-        throw new Error('Unknown type');
-    }
-  })
-
-  const todo = status.todo.reduce(reducer);
-  const pr = status.PR.reduce(reducer);
-  const done = status.done.reduce(reducer);
-  const total = todo + pr + done;
+  const statues = report.reduce((statues, report) => {
```
Comment (timja) [inline reports.js]:
```suggestion
  const statuses = report.reduce((statues, report) => {
```

Comment (halkeye) [reply reports.js]: yea I can make that change everywhere

**Pull request merged**